File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/ord/dto/d/OrSuModRtnInfoDTO.java
package iih.ci.ord.dto.d;

import xap.mw.core.data.*;
import xap.mw.coreitf.d.*;
import java.math.BigDecimal;

/**
 * 医嘱状态修改返回信息 DTO数据 
 * 
 */
public class OrSuModRtnInfoDTO extends BaseDTO {
	private static final long serialVersionUID = 1L;
	
	/**
	 * 主键
	 * @return String
	 */
	public String getId() {
		return ((String) getAttrVal("Id"));
	}
	/**
	 * 主键
	 * @param Id
	 */
	public void setId(String Id) {
		setAttrVal("Id", Id);
	}
	/**
	 * 业务时间
	 * @return FDateTime
	 */
	public FDateTime getDt_biz() {
		return ((FDateTime) getAttrVal("Dt_biz"));
	}
	/**
	 * 业务时间
	 * @param Dt_biz
	 */
	public void setDt_biz(FDateTime Dt_biz) {
		setAttrVal("Dt_biz", Dt_biz);
	}
	/**
	 * 科室
	 * @return String
	 */
	public String getId_dept() {
		return ((String) getAttrVal("Id_dept"));
	}
	/**
	 * 科室
	 * @param Id_dept
	 */
	public void setId_dept(String Id_dept) {
		setAttrVal("Id_dept", Id_dept);
	}
	/**
	 * 科室编码
	 * @return String
	 */
	public String getCode_dept() {
		return ((String) getAttrVal("Code_dept"));
	}
	/**
	 * 科室编码
	 * @param Code_dept
	 */
	public void setCode_dept(String Code_dept) {
		setAttrVal("Code_dept", Code_dept);
	}
	/**
	 * 科室名称
	 * @return String
	 */
	public String getName_dept() {
		return ((String) getAttrVal("Name_dept"));
	}
	/**
	 * 科室名称
	 * @param Name_dept
	 */
	public void setName_dept(String Name_dept) {
		setAttrVal("Name_dept", Name_dept);
	}
	/**
	 * 人员
	 * @return String
	 */
	public String getId_emp() {
		return ((String) getAttrVal("Id_emp"));
	}
	/**
	 * 人员
	 * @param Id_emp
	 */
	public void setId_emp(String Id_emp) {
		setAttrVal("Id_emp", Id_emp);
	}
	/**
	 * 人员编码
	 * @return String
	 */
	public String getCode_emp() {
		return ((String) getAttrVal("Code_emp"));
	}
	/**
	 * 人员编码
	 * @param Code_emp
	 */
	public void setCode_emp(String Code_emp) {
		setAttrVal("Code_emp", Code_emp);
	}
	/**
	 * 人员名称
	 * @return String
	 */
	public String getName_emp() {
		return ((String) getAttrVal("Name_emp"));
	}
	/**
	 * 人员名称
	 * @param Name_emp
	 */
	public void setName_emp(String Name_emp) {
		setAttrVal("Name_emp", Name_emp);
	}
}

File: client/iih_stub/iih.ci_stub/iih.ci.ord/dto/d/OrSuModRtnInfoDTO.cs
using System;
using xap.mw.core.data;
using xap.mw.coreitf.d;

namespace iih.ci.ord.dto.d
{
    /// <summary>
    /// OrSuModRtnInfoDTO 的摘要说明。
    /// </summary>
    public class OrSuModRtnInfoDTO : BaseDTO {

        public OrSuModRtnInfoDTO() {
 
        }

        /// <summary>
        /// 主键
        /// </summary>
		public string Id {
            get { return getAttrVal<string>("Id",null); }
            set { setAttrVal<string>("Id", value); }
        }

        /// <summary>
        /// 业务时间
        /// </summary>
		public DateTime? Dt_biz {
            get { return getAttrVal<FDateTime>("Dt_biz",null); }
            set { setAttrVal<FDateTime>("Dt_biz", value); }
        }

        /// <summary>
        /// 科室
        /// </summary>
		public string Id_dept {
            get { return getAttrVal<string>("Id_dept",null); }
            set { setAttrVal<string>("Id_dept", value); }
        }

        /// <summary>
        /// 科室编码
        /// </summary>
		public string Code_dept {
            get { return getAttrVal<string>("Code_dept",null); }
            set { setAttrVal<string>("Code_dept", value); }
        }

        /// <summary>
        /// 科室名称
        /// </summary>
		public string Name_dept {
            get { return getAttrVal<string>("Name_dept",null); }
            set { setAttrVal<string>("Name_dept", value); }
        }

        /// <summary>
        /// 人员
        /// </summary>
		public string Id_emp {
            get { return getAttrVal<string>("Id_emp",null); }
            set { setAttrVal<string>("Id_emp", value); }
        }

        /// <summary>
        /// 人员编码
        /// </summary>
		public string Code_emp {
            get { return getAttrVal<string>("Code_emp",null); }
            set { setAttrVal<string>("Code_emp", value); }
        }

        /// <summary>
        /// 人员名称
        /// </summary>
		public string Name_emp {
            get { return getAttrVal<string>("Name_emp",null); }
            set { setAttrVal<string>("Name_emp", value); }
        }
        /// <summary>
        /// 返回属性列表
        /// </summary>
        /// <returns></returns>
        public override string[] getAttrNames()
        {
            return new string[]{ "Id", "Dt_biz", "Id_dept", "Code_dept", "Name_dept", "Id_emp", "Code_emp", "Name_emp"};
        }
        
        /// <summary>
        /// 返回全路径DO类名
        /// </summary>
        /// <returns></returns>
        public override string getFullClassName()
        {
            return "iih.ci.ord.dto.d.OrSuModRtnInfoDTO";
        }
    }
}
